.github/workflows/package-pusher.yml

```
@@ -0,0 +1,59 @@
+name: Package Pusher
+
+on:
+  workflow_dispatch:
+    inputs:
+      packages:
+        description: "What is the packages you wish to push (seperate package names with a whitespace)?"
+        required: true
+
+jobs:
+  pusher:
+    runs-on: windows-2019
+    env:
+      github_user_repo: ${{ github.repository }}
+      API_KEY: ${{ secrets.CHOCO_API_KEY }}
+      SOURCE_LOCATION: "https://push.chocolatey.org"
+
+    steps:
+      - name: Install Dependencies
+        run: |
+          # Placeholder for installing dependencies
+          # choco install chocolatey-core.extension -y
+        shell: powershell
+      - name:
+        run: |
+          Get-CimInstance win32_operatingsystem -Property Caption, OSArchitecture, Version | fl Caption, OSArchitecture, Version
+          $PSVersionTable
+          choco --version
+        shell: powershell
+      - uses: actions/checkout@v3.0.0
+      - name: Pack Packages
+        env:
+          PACKAGE_PUSHES: ${{ github.event.inputs.packages }}
+        run: |
+          $packages = $env:PACKAGES_PUSHES -split ' '
```

```suggestion
          $packages = $env:PACKAGE_PUSHES -split ' '
```

Or the line above needs to be `PACKAGES_PUSHES`

---

That is what I get when doing a partial copy from my own repository where I use a similar action.

Good catch.